scratch-vm/src/extensions/scratch3_tello/index.js

```diff
@@ -25,216 +25,248 @@ const message = {
         'ja-Hira': 'りりくする',
         'en': 'takeoff',
         'ru': 'взлёт',
-        'fr': 'décollage'
+        'fr': 'décollage',
+        'de': 'abheben'
     },
     land: {
         'ja': '着陸する',
         'ja-Hira': 'ちゃくりくする',
         'en': 'land',
         'ru': 'посадка',
-        'fr': 'atterrissage'
+        'fr': 'atterrissage',
+        'de': 'landen'
     },
     up: {
         'ja': '上に [X]cm 上がる',
         'ja-Hira': 'うえに [X] センチあがる',
         'en': 'up [X] cm',
         'ru': 'вверх [X] см',
-        'fr': 'montée de [X] cm'
+        'fr': 'montée de [X] cm',
+        'de': '[X] cm höher'
     },
     down: {
         'ja': '下に [X]cm 下がる',
         'ja-Hira': 'したに [X] センチさがる',
         'en': 'down [X] cm',
         'ru': 'вниз [X] см',
-        'fr': 'descente de [X] cm'
+        'fr': 'descente de [X] cm',
+        'de': '[X] cm tiefer'
     },
     left: {
         'ja': '左に [X]cm 動く',
         'ja-Hira': 'ひだりに [X] センチうごく',
         'en': 'move left [X] cm',
         'ru': 'влево [X] см',
-        'fr': 'voler à gauche [X] cm'
+        'fr': 'voler à gauche [X] cm',
+        'de': '[X] cm nach links'
     },
     right: {
         'ja': '右に [X]cm 動く',
         'ja-Hira': 'みぎに [X] センチうごく',
         'en': 'move right [X] cm',
         'ru': 'вправо [X] см',
-        'fr': 'voler à droite [X] cm'
+        'fr': 'voler à droite [X] cm',
+        'de': '[X] cm nach rechts'
     },
     forward: {
         'ja': '前に [X]cm 進む',
         'ja-Hira': 'まえに [X] センチすすむ',
         'en': 'move forward [X] cm',
         'ru': 'вперёд [X] см',
-        'fr': 'voler vers l\'avant [X] cm'
+        'fr': 'voler vers l\'avant [X] cm',
+        'de': '[X] cm nach vorne'
     },
     back: {
         'ja': '後ろに [X]cm 下がる',
         'ja-Hira': 'うしろに [X] センチさがる',
         'en': 'move back [X] cm',
         'ru': 'назад [X] см',
-        'fr': 'voler vers l\'arrière [X] cm'
+        'fr': 'voler vers l\'arrière [X] cm',
+        'de': '[X] cm nach hinten'
     },
     cw: {
         'ja': '[X] 度右に回る',
         'ja-Hira': '[X] どみぎにまわる',
         'en': 'rotate [X] degrees right',
         'ru': 'повернуть на [X] градусов вправо',
-        'fr': 'tourner de [X] degrés vers la droite'
+        'fr': 'tourner de [X] degrés vers la droite',
+        'de': 'drehe [X] Grad nach rechts'
     },
     ccw: {
         'ja': '[X] 度左に回る',
         'ja-Hira': '[X] どひだりにまわる',
         'en': 'rotate [X] degrees left',
         'ru': 'повернуть на [X] градусов влево',
-        'fr': 'tourner de [X] degrés vers la gauche'
+        'fr': 'tourner de [X] degrés vers la gauche',
+        'de': 'drehe [X] Grad nach links'
     },
     flip: {
         'ja': '[DIRECTION]に宙返りする',
         'ja-Hira': '[DIRECTION]にちゅうがえりする',
         'en': 'flip in [DIRECTION]',
-        'fr': 'flip vers [DIRECTION]'
+        'fr': 'flip vers [DIRECTION]',
+        'de': 'Flip nach [DIRECTION]'
```

Comment:
Perhaps the first letter is lowercase?
(If "Flip" is used as a noun and is capitalized, ignore my review.)
```suggestion
        'de': 'flip nach [DIRECTION]'
```

Reply:
I would really use it as a noun in this context. I also thought of "Drehung" but this rather means a circular movement on the x-axis and would thus be the wrong translation. Thanks for checking it after such a short time.

Reply:
Understood. Now, I'm going to merge.